### ISSUE TITLE
missing ExcludeFocus

### ISSUE THREAD

**Issue by GuoDapeng** [flutter/gallery#152]
**Describe the bug**
The cloned project cannot be run directly, missing ExcludeFocus

**Flutter doctor**
```
 ~  flutter doctor -v
[✓] Flutter (Channel stable, v1.17.0, on Mac OS X 10.15.4 19E287, locale
    zh-Hans-CN)
    • Flutter version 1.17.0 at /Users/felix/Documents/sdk/flutter
    • Framework revision e6b34c2b5c (5 days ago), 2020-05-02 11:39:18 -0700
    • Engine revision 540786dd51
    • Dart version 2.8.1


[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/felix/Library/Android/sdk
    • Platform android-R, build-tools 29.0.3
    • Java binary at: /Users/felix/Library/Application
      Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/192.6392135/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.9.1

[✓] Android Studio (version 3.6)
    • Android Studio at /Users/felix/Library/Application
      Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/192.6392135/Android
      Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)

[✓] Android Studio
    • Android Studio at /Users/felix/Library/Application
      Support/JetBrains/Toolbox/apps/AndroidStudio/ch-1/193.6423924/Android
      Studio 4.1 Preview.app/Contents
    • Flutter plugin version 45.1.2
    • Dart plugin version 193.6911.31
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[!] IntelliJ IDEA Ultimate Edition (version 2020.1.1)
    • IntelliJ at /Users/felix/Applications/JetBrains Toolbox/IntelliJ IDEA
      Ultimate.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[!] Connected device
    • Error: GDP iPad Pro is not connected. Xcode will continue when GDP iPad
      Pro is connected. (code -13)
    • Error: 郭大鹏的iPhone is not connected. Xcode will continue when 郭大鹏的iPhone is
      connected. (code -13)

! Doctor found issues in 2 categories.
```

**To Reproduce**
1. Clone the project
2. flutter pub get
3. found mistake

**Expected behavior**
Able to run

**Screenshots**
![截屏2020-05-08 上午10 28 23](https://user-images.githubusercontent.com/7477750/81363951-191fa100-9117-11ea-8ccb-045b1a0e5a8b.png)

**Additional context**
Sorry, my bad English, try to understand the picture

**Comment by guidezpl** [comment]
I see you're using the stable channel. The Gallery runs off the master channel to support new features (like `Exclude Focus`) that haven't made it into a stable release yet.

You can use
```bash
flutter channel master
flutter upgrade
```

to switch channels.

**Comment by GuoDapeng** [comment]
Sorry, I didn't understand the documentation. I can work now.

**Comment by dzhang1987** [comment]
> Sorry, I didn't understand the documentation. I can work now.

大佬，你怎么搞好的？我遇到同样的问题。

**Comment by sm9i** [comment]
> > 抱歉，我不了解文档。我现在可以工作。
> 
> 大佬，你怎么搞好的？我遇到同样的问题。

切换到master就ok了

**Comment by GuoDapeng** [comment]
> > Sorry, I didn't understand the documentation. I can work now.
> 
> 大佬，你怎么搞好的？我遇到同样的问题。

https://github.com/flutter/gallery/issues/152#issuecomment-625749310

按照他说的，弄好了，管用。ExcludeFocus 这个是新出的。

**Comment by mshwf** [comment]
I have this issue and I'm on master channel (Channel master, 1.19.0-2.0.pre.133)

**Comment by JSWilProf** [comment]
This widget is in the "packages/flutter/lib/src/widgets/focus_scope.dart" file and was included in the commit fdc4d21b7 - Flutter 1.18.0-10.0.pre

**Comment by mshwf** [comment]
@JSWilProf I opened that file in VSCode (in the same project window), and somehow the project could find the missing reference!
